core/src/skybox.cpp

```diff
@@ -0,0 +1,166 @@
+#include "knoting/skybox.h"
+
+namespace knot {
+namespace components {
+
+struct PosColorTexCoord0Vertex {
+    float m_x;
+    float m_y;
+    float m_z;
+    uint32_t m_rgba;
+    float m_u;
+    float m_v;
+
+    static void init() {
+        ms_layout.begin()
+            .add(bgfx::Attrib::Position, 3, bgfx::AttribType::Float)
+            .add(bgfx::Attrib::Color0, 4, bgfx::AttribType::Uint8, true)
+            .add(bgfx::Attrib::TexCoord0, 2, bgfx::AttribType::Float)
+            .end();
+    }
+
+    static bgfx::VertexLayout ms_layout;
+};
+
+bgfx::VertexLayout PosColorTexCoord0Vertex::ms_layout;
+static float s_texelHalf = 0.0f;
```

Comment:
Can this be a `constexpr`? If not that is okay, just checking, since nothing else appears to be using it.

Reply:
unused variable from screen space quad